### PR TITLE
Refactor cmd structure

### PR DESCRIPTION
**Pull request by tpantelis**
- Eliminated the armada/cmd/armada stutter
- Combined repeated modules, eg cmd/create/cluster and cmd/destroy/cluster -> cmd/cluster
- Renamed functions appropriately to avoid stutter
- Made internal structs package-scoped

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

### CODE REVIEW

File: cmd/cluster/create.go [hunk 1]
```diff
@@ -17,53 +17,53 @@ import (
...
 	Tiller bool
 
-	// Overlap if to create clusters with overlapping cidrs
+	// Whether or not to create clusters with overlapping cidrs
 	Overlap bool
 
-	// Debug sets log level to debug
-	Debug bool
+	// Log level to debug
+	debug bool
```
Comment (dfarrell07) [inline cmd/cluster/create.go]: Per question/discussion with @tpantelis on Slack, might want to make this debug flag exported (Debug), parallel to the other flags here.

Comment (tpantelis) [reply cmd/cluster/create.go]: Done

File: test/e2e/e2e_test.go [hunk 2]
```diff
@@ -358,21 +340,20 @@ var _ = Describe("E2E Tests", func() {
...
-				Debug:  true,
-				Images: []string{"nginx:stable-alpine", "alpine:latest"},
-				Clusters: []string{
-					defaults.ClusterNameBase + strconv.Itoa(1),
-					defaults.ClusterNameBase + strconv.Itoa(3),
-				},
-			}
 			log.SetLevel(log.DebugLevel)
+
+		    images := []string{"nginx:stable-alpine", "alpine:latest"}
```
Comment (dfarrell07) [inline test/e2e/e2e_test.go]: I think this line should be indented one more level.

**Comment by tpantelis** [comment]
> One minor indentation error noted inline, and some import ordering errors that seem to be introduced by this change.
> 

I ran `golangci-lint run` but no errors were reported. The imports were re-ordered to be alphabetical by `goimports`.

File: test/e2e/e2e_test.go (hunk 2 above)
Comment (tpantelis) [reply test/e2e/e2e_test.go]: Done. It looked correct in GoLand - I guess one of the indentations must have had spaces and not a tab.

**Comment by tpantelis** [comment]
> Also, when I build the binary from this patch (but not the main armada-review branch), I get this error when creating clusters:
> 
> ```
> $ ./armada create clusters
> FATA[2020-01-22 07:56:29] cluster2: stat /home/daniel/go/src/github.com/submariner-io/configs/tpl/cluster-config.yaml: no such file or directory 
> ```

Fixed. **root.go**  was moved so it's relative path to _configs_ changed from _../../configs_ to _../configs_.

**Comment by dfarrell07** [comment]
> > One minor indentation error noted inline, and some import ordering errors that seem to be introduced by this change.
> 
> I ran `golangci-lint run` but no errors were reported. The imports were re-ordered to be alphabetical by `goimports`.

Okay, it works for me. FYI you can see the difference in the builds I'm referring to starting on L582 of both the [latest build of the base branch without this commit](https://travis-ci.com/submariner-io/armada/jobs/278358781#L582) and the [latest build with this commit](https://travis-ci.com/submariner-io/armada/jobs/279299936#L582).

**Comment by tpantelis** [comment]
> Okay, it works for me. FYI you can see the difference in the builds I'm referring to starting on L582 of both the [latest build of the base branch without this commit](https://travis-ci.com/submariner-io/armada/jobs/278358781#L582) and the [latest build with this commit](https://travis-ci.com/submariner-io/armada/jobs/279299936#L582).

ahh - this was before I changed my Goland to run goimports on save. I pushed https://github.com/submariner-io/armada/pull/4/commits/d90800170e26f1a5f0d6c4bd3307192a0fa74a59 to fix the complaints.

**Pull request merged**